projects/mysql-server/build.sh

```diff
@@ -27,6 +27,8 @@ else
 fi
 make install
 cp $OUT/mysql/bin/fuzz* $OUT/
+#FIXME: Temporary disabled due to fix needed in target with mysql 8.21
+#rm $OUT/fuzz_docommand
```

Comment:
I dont understand this, why do we need this here, when the rm is commented out. Can you remove these two lines.